### DISCUSSION TITLE
Pressure advance help

### DISCUSSION THREAD

**Discussion by Edzi11** [OrcaSlicer/OrcaSlicer#238] [Q&A]
I'm currently using this fork on my ender 3 clone. Trying to use the pressure advance line and tower test to find a good K value. For the life of me I cannot see any difference on a tower or line test. I noticed that on the line test the fast and slow values are too low. 

According to the preview the slow section is 20 MMS while the fast is 27 which isn't a huge variation so it's hard to tell the differences. While doing the line test via Marlin website I am getting actual variations but the issue is that I cannot repeat this test via the slicer due to modern website having stock settings. 

I usually print with .32 layer width to get more detail and smoother top finish and I also run 90% flow. With a Martin website I can't seem to input these settings and when I try to input these settings in this slicer it seems to go mute as the speed variations pretty much cancel out any beneficial results. Also according to the mom and website my K value was as high as .16 on direct drive but when doing actual prints I'm getting some pretty gnarly results. 

Attached or some pictures of pressure advanced test done on default in the slicer. 
![20230124_174314](https://user-images.githubusercontent.com/123626106/214789710-1794514a-07c6-4316-87e8-904324a0b817.jpg)
![20230125_101609](https://user-images.githubusercontent.com/123626106/214789719-4d7f87a7-01bc-4dcb-914b-e932faa4e732.jpg)
![20230125_101556](https://user-images.githubusercontent.com/123626106/214789725-a7ddf572-ebcf-4699-8c40-8f2a98b5e897.jpg)
![20230125_101547](https://user-images.githubusercontent.com/123626106/214789731-a54ec21c-892f-4453-93ef-b52cdecc454a.jpg)

**Comment by rich0** [comment]
This didn't get a reply, and I am very new to this, but I had a similar issue with PETG on my X1C for the tower (not the line) test in version 1.4.3.  I noticed that version 1.4.4 had fixes to the tower in the release notes, and the tower had more obvious impacts from the PA setting in this version.  So if you haven't done so, I'd suggest repeating this in v1.4.4.

**Reply by Edzi11** [reply]
Thanks for the suggestion! Honestly was really hoping this thread would be more active but I guess not. I'll give that a shot though

**Comment by SoftFever** [comment]
A UI bug in 1.4.3 prevented PA tower calibration work in some cases. It is fixed in 1.4.4.

**Reply by Edzi11** [reply]
Okay cool I thought maybe I was doing something wrong, any chance we can get a configurable pa calibration?

**Reply by SoftFever** [reply]
Yes😀It’s already configurable in the latest release (1.4.4)

**Reply by Edzi11** [reply]
Oh I haven't checked it out yet my bad but thank you! You're awesome! Just received my X1 today, any suggestions in mods or if there's anything I should change in the starting or ending G-Code?

**Reply by SoftFever** [reply]
Use it as is for few weeks I’d recommend 🙂

**Reply by Brett19E7** [reply]
Hey guys I'm stuck I get an error when I click the pressure test puzzle and tower but not the lines why is this?

**Comment by Brett19E7** [comment]
![PXL_20231125_162241693](https://github.com/SoftFever/OrcaSlicer/assets/152027871/80482253-3fd2-4626-932e-024e3b4f0cf5)
![da3acc01-e4fe-46cd-983e-8c245421fad1](https://github.com/SoftFever/OrcaSlicer/assets/152027871/1d316fd3-9782-4e0b-b236-70824b171700)
